core/domain/exp_jobs_one_off_test.py

```diff
@@ -1242,17 +1242,8 @@ def test_exp_state_pairs_are_produced_only_for_desired_interactions(self):
             'tagged_misconception_id': None
         }]
 
-        content_ids_to_audio_translations_dict = {
```

Comment:
I see a lot of removals, will we have replacement tests for `recorded_voiceovers`?